test/unit/AccountInfo.js

```diff
@@ -0,0 +1,83 @@
+import AccountInfo from "../src/account/AccountInfo.js";
+import AccountId from "../src/account/AccountId.js";
+import Hbar from "../src/Hbar.js";
+import { Key } from "@hashgraph/proto";
+import Timestamp from "../src/Timestamp.js";
+import Duration from "../src/Duration.js";
+import TokenRelationshipMap from "../src/account/TokenRelationshipMap.js";
+import Long from "long";
+
+/**
+ *  sendRecordThreshold and receiveRecordThreshold are deprecated and not included in this test nor toJSON
+ */
+
+describe("AccountInfo", function () {
+    it("should construct AccountInfo JSON object", function () {
+        const props = {};
+        props.accountId = new AccountId(1, 1, 1);
+        props.contractAccountId = "0000000000000000000000000000000000000001";
+        props.isDeleted = false;
+        props.proxyAccountId = "0000000000000000000000000000000000000002";
+        props.proxyReceived = new Hbar(1);
+        props.key = new Key();
+        props.balance = new Hbar(1);
+        props.isReceiverSignatureRequired = false;
+        props.expirationTime = new Timestamp(1, 1);
+        props.autoRenewPeriod = new Duration(1);
+        props.liveHashes = [];
+        props.tokenRelationships = new TokenRelationshipMap();
+        props.accountMemo = "This is a memo";
+        props.ownedNfts = new Long(1);
+
+        const accountInfo = new AccountInfo(props).toJSON();
+
+        expect(accountInfo["accountId"]).to.eql("1.1.1");
```

Comment:
What I'd recommend is you implement `AccountInfo.toString()` and `AccountInfo.fromString()` then you can do
```javascript
const accountInfo = AccountInfo.fromString(new AccountInfo(props).toString());
```